tests/data/adaptive-concurrency/shallow-knee.toml

```diff
@@ -12,12 +12,12 @@ mean = [9, 11]
 mode = [9, 12]
 
 [stats.rate]
-max = [108, 108]
-mean = [87, 88]
+max = [103, 108]
+mean = [87, 89]
```

Comment:
I think this is our one regression, that this one test showed a maximum rate decrease. However, the mean showed a trivial increase, so the overall performance is still up a hair.